src/components/invoices/InvoicesList.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component, Fragment } from 'react'
+import InvoiceItem from './InvoiceItem'
+import Spinner from '../layout/spinner/Spinner'
+import TopBar from '../layout/topBar/TopBar'
+import Table from '../layout/table/Table'
+import InvoiceDetails from './InvoiceDetails/InvoiceDetails'
+import './invoices.css'
+
+export class InvoicesList extends Component {
+    state = {
+        invoices: [],
+        loading: false,
+        current: null,
+        contractorInfo: null
+    }
+
+    componentDidMount() {
+        this.setState({ loading: true })
+        fetch('http://localhost:5000/invoices')
+            .then(res => res.json())
```

Comment:
you can skip this `this.setState({ loading: true })` in cDM - that makes unnecessary render. Set you default state to `{..., loading: true, ...}`